webpack.config.js

```diff
@@ -25,6 +25,7 @@ const SHARED_COMP_DIR = path.resolve(__dirname, 'app/shared-components');
 const PANEL_DIR = path.resolve(__dirname, 'app/panel');
 const PANEL_ANDROID_DIR = path.resolve(__dirname, 'app/panel-android');
 const HUB_DIR = path.resolve(__dirname, 'app/hub');
+const HUB_GHOSTERY_BROWSER_DIR = path.resolve(__dirname, 'app/ghostery-browser-hub');
```

Comment:
@wlycdgr Looks like some places we use `ghostery-browser-hub` and others `dawn-onboarding`. I think we should make the directories and names consistent.

Reply:
maybe just `dawn-hub` since the other folders are `hub` and `shared-hub`?

Reply:
The code has been updated to consistently use `dawn-hub`, Dawn Hub, `dawn_hub`, etc as appropriate.

The one exception I'm aware of is in the images folders, but those also require some reorganization, so I put them aside for the moment.